_dev/.stylelintrc

```diff
@@ -16,16 +16,20 @@
       true,
       {
         "ignoreAtRules": [
-          "tailwind",
+          "at-root",
+          "each",
+          "else",
+          "error",
+          "extend",
           "function",
           "if",
-          "each",
           "include",
+          "layer",
           "mixin",
-          "use",
-          "extend",
+          "return",
           "screen",
-          "layer"
+          "tailwind",
+          "use"
```

Comment:
Sorry, I added a couple new `@` rules here and then sorted A-z, hence the extra diff noise. #a2z4life